packages/bootstrap/lib/globals.module.scss

```diff
@@ -0,0 +1,13 @@
+/**
+ * IMPORTANT
+ * Before adding styles to this class, think _very carefully_ about
+ * possible side effects this may have on apps. These styles will
+ * be applied to all apps that consume Kaizen
+ */
+
+/** 
+ * Component-level overrides 
+ */
+[data-kaizen-component] {
+  box-sizing: border-box;
```

Comment:
For this specific case, do we need to opt all descendants into this rule too?

Reply:
Yeah, 🤔 I would hope that if an engineer is opting out of these component defaults then they should set reasonable defaults that work for them to stop this cascade. That's not really clear from this though. For clarity this feels like the expected behaviour to me:
https://codepen.io/actuallyacat/pen/abJwGZR?editors=1100